### ISSUE TITLE
Removal of a linked local backend to a global backend makes the backend(s) page unable to fully load

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework-alba-plugin#410]
### Problem description
Removal of a local backend which has been linked to a global backend, makes both the global backend page as the backends page unable to fully load

Backends page:
```
https://10.100.199.201/api/backends/?page=1&page_size=10&progressive=false&sort=name&contents=_relations%2Cregular_domains%2Clive_status&timestamp=1496149343149
{error_description: "live_status", error: "internal_server"}
```
Global backend page:
```
ttps://10.100.199.201/api/backends/0c4e32e0-1bdb-4b22-b3e1-c5ba59f2fd09/?contents=_relations%2Cregular_domains%2Caccess_rights%2Clive_status&timestamp=1496149394453
{error_description: "live_status", error: "internal_server"}
```

### Steps that I undertook
- Create two local backends
- Claimed those asds 
- Linked both to a global backend
- Removed asds on one local local backend
- Saw the issue
- Removed all asds on the other local local backend
- Still perceived the issue
- Cried in a corner

Logs:
There is no direct error to be found in the worker logs but here they are:
[logs_backend_removal.txt](https://github.com/openvstorage/framework-alba-plugin/files/1038425/logs_backend_removal.txt)

**Comment by khenderick** [comment]
Can you add the logs of the internal server error?

**Comment by JeffreyDevloo** [comment]
The error is related to the live_status query.
Getting the variable myself gives off:
```
In [6]: for b in AlbaBackendList.get_albabackends():
    print b.live_status
   ...:     
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-6-691081103c91> in <module>()
      1 for b in AlbaBackendList.get_albabackends():
----> 2     print b.live_status
      3 

/opt/OpenvStorage/ovs/dal/dataobject.pyc in <lambda>(s)
    342         """
    343         # pylint: disable=protected-access
--> 344         fget = lambda s: s._get_dynamic_property(dynamic)
    345         # pylint: enable=protected-access
    346         setattr(self.__class__, dynamic.name, property(fget))

/opt/OpenvStorage/ovs/dal/dataobject.pyc in _get_dynamic_property(self, dynamic)
    411         """
    412         data_loader = getattr(self, '_{0}'.format(dynamic.name))
--> 413         return self._backend_property(data_loader, dynamic)
    414 
    415     # Helper method supporting property setting

/opt/OpenvStorage/ovs/dal/dataobject.pyc in _backend_property(self, function, dynamic)
   1001                         dynamic_data = function(dynamic=dynamic)  # Load data from backend
   1002                     else:
-> 1003                         dynamic_data = function()
   1004                     correct, allowed_types, given_type = DalToolbox.check_type(dynamic_data, dynamic.return_type)
   1005                     if not correct:

/opt/OpenvStorage/ovs/dal/hybrids/albabackend.pyc in _live_status(self)
    518         linked_backend_warning = False
    519         for remote_info in self.remote_stack.itervalues():
--> 520             if remote_info['error'] == 'unknown' or remote_info['live_status'] == 'failure':
    521                 return 'failure'
    522             if remote_info['error'] == 'not_allowed':

KeyError: 'live_status'

```

**Comment by khenderick** [comment]
With no decent logging there isn't much I can do to reproduce or investigate this. However, a small staring contest with the code did reveal a possible issue that would give the same stacktrace. I'll fix that one.

**Comment by khenderick** [comment]
Fixed by #426, packaged in openvstorage-backend-1.9.2-dev.1497432293.ba4e853

**Comment by JeffreyDevloo** [comment]
openvstorage-backend_1.9.2-1_amd64.deb
